src/pages/git/git-checkout/index.md

```diff
@@ -3,13 +3,38 @@ title: Git Checkout
 ---
 ## Git Checkout
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/git/git-checkout/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Git Checkout switches between branches or restore working tree files.  There are a lot of options to checkout that won't be covered here but you can take a look at all of them [here][checkout-options] 
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+### Checkout a Branch
+To checkout an existing branch, run the command:
+```shell
+git checkout BRANCH-NAME
+```
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+Generally, Git won't let you checkout another branch unless your working directory is clean, because you would lose any working directory changes that aren't committed. You have three options to handle your changes: 1) trash them, 2) commit them, or 3) stash them.
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+### Checkout New Branch
+To checkout out and create a new branch, run the command:
+```shell
+git checkout -b NEW-BRANCH-NAME
+```
+This will automatically switch you to the new branch
 
+### Check out New Branch or Reset to Start Point
+Creates the branch NEW-BRANCH and start it at <start_point>; if it already exists, then reset it to <start_point>. This is equivalent to running "git branch" with "-f"
```

Comment:
So, we had an older issue where these angle brackets were rendering as HTML elements (that the browser didn't know what to do with, so the article looked odd). That's been fixed, but we've since adopted the all-caps with hyphens (as you noticed with NEW-BRANCH 👍 ). For this sentence, wrap "NEW-BRANCH"/ the "git branch" and "-f" references in backticks (don't need the quotes for the latter ones), and change "<start_point>" references to `START-POINT`.